.mise.toml

```
@@ -7,7 +7,7 @@ AWS_PROFILE = "oph-ktr-dev"
 node = '23.2.0'
 "npm:prettier" = "3.3.3"
 maven = "3.9.9"
-awscli = "ref:2.20.0"
+awscli = "ref:2.21.0"
```

Uusin versio on nyt 2.21.2
https://github.com/aws/aws-cli/tags

```suggestion
awscli = "ref:2.21.2"
```